src/Build.UnitTests/BackEnd/SdkResolverLoader_Tests.cs

```diff
@@ -32,8 +32,8 @@ public void AssertDefaultLoaderReturnsDefaultResolver()
 
             var resolvers = loader.LoadResolvers(_loggingContext, new MockElementLocation("file"));
 
-            Assert.Equal(1, resolvers.Count);
-            Assert.Equal(typeof(DefaultSdkResolver), resolvers[0].GetType());
+            resolvers.Count.ShouldBe(2);
+            resolvers[1].GetType().ShouldBe(typeof(DefaultSdkResolver));
```

Comment:
Also check `resolvers[0]`?

Reply:
The unit test is only verifying that `DefaultSdkResolver` was added.  Should I check for both here or make a new test?

Reply:
I'd change it to `AssertDefaultLoaderReturnsDefaultResolver`**`s`** and check for both here.